test/hash_test.rb

```diff
@@ -3,6 +3,8 @@
 require 'hanami/utils/hash'
 
 describe Hanami::Utils::Hash do
+  CURRENT_RUBY_VERSION = defined?(TRAVIS_RUBY_VERSION) ? TRAVIS_RUBY_VERSION : RUBY_VERSION
```

Comment:
Do you mean they ship a patched version of Ruby where `RUBY_VERSION` isn't available?